packages/rest/README.md

```diff
@@ -120,7 +120,7 @@ Starting your own server is more involved than using the CLI, but allows more fi
 To get started, make sure you have set up the correct version of Aries Framework JavaScript according to the AFJ repository. To find out which version of AFJ you need to have installed you can run the following command. This will list the required peer dependency for `@aries-framework/core`.
```

Comment:
I think we can remove this whole section + the code snippet, as the rest package will now install the AFJ core version it needs

```suggestion
```